python/tests/res/config/test_config.py

```diff
@@ -91,7 +91,10 @@ def test_item_types(self):
             self.assertEqual( path_value , "file")
             self.assertEqual( type_item.igetString(4) , "file")
 
-
+            # test __getitem__
+            self.assertTrue(conf['TYPE_ITEM'])
```

Comment:
Yes, you could say that, but also that  `__getitem__` is implemented, and that it returns a truthy type for the existing key `TYPE_ITEM`.  Should I remove the comment?